README.md

```diff
@@ -82,6 +82,48 @@ Definitions of the flags used in the below example
 swagger-codegen generate -l rails5 -i public/doc/swagger-2.yaml -o /tmp/sp -t swagger-codegen-templates/'
 ```
 
+## Testing with Dev Insights UI
+
+If you want to run the service portal locally on your dev machine but access the UI on the dev cluster you would need to do the following
+
+Install Docker in your machine
+
+Clone the following repo to your dev machine
+
+[Insights Proxy](https://github.com/RedHatInsights/insights-proxy)
+
+Follow the instructions in [README.md](https://github.com/RedHatInsights/insights-proxy/blob/master/README.md)
+
+
+In the service-portal repository the dev_configs directory contains 2 JavaScript files which can be used with insights proxy to route the service portal requests to your dev machine
+
+1. service-portal_linux.js
+
+2. service-portal_mac.js
+
+The insights proxy runs a docker container and it can be tailored using config files
+
+You would need 2 terminals for this setup
+
+1. **Running your service portal app**
+
+      export APP_NAME=service-portal
+      
+      export PATH_PREFIX=/r/insights/platform
+      
+      bin/rails s -p 5000
+      
+2. **Run the insights proxy based on Linux or Mac**
+```
+   SPANDX_CONFIG=/path/to/service_portal_api/dev_configs/. service-portal_mac.js service bash /path/to/insights-proxy/scripts/run.sh
```

Comment:
For the Mac I ended up using: 
```
SPANDX_CONFIG=/path/to/service_portal_api/dev_configs/service-portal_mac.js bash /path/to/insights-proxy/scripts/run.sh
```
Dropping the ". " after `dev_config/` and also removed `service` as both caused `command not found` errors.